style.css

```diff
@@ -0,0 +1,92 @@
+body {
+  width: 375px;
+  height: 590px;
+}
+
+header {
+  background-color: #3c3a39;
+  display: flex;
+  height: 48px;
+}
+
+.logo {
+  width: 77px;
+  height: 24px;
+  font-family: sans-serif, 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+  text-align: center;
+  padding-top: 15px;
+}
+
+.image {
+  padding-top: 23px;
+  padding-left: 250px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  height: 590px;
+  width: 375px;
+  border-radius: 0;
+  background: #1c1a19;
+}
+
+h2 {
+  font-family: sans-serif, 'Round';
+  font-size: 50px;
+  font-weight: 400px;
+  color: #ff6b00;
+  height: 124px;
+  padding-top: 10px;
+  padding-left: 20px;
+  width: 327px;
+}
+
+p {
+  width: 359px;
+  height: 27px;
+  font-family: sans-serif, 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  color: #fff5e1;
+  padding-left: 20px;
+}
+
+.description {
+  width: 327px;
+  height: 123px;
+  font-family: sans-serif, 'Poppins';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding: 20px;
+  color: #fff;
+}
+
+.socialmedia {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 20px;
+  padding-bottom: 40px;
+  width: 200px;
+  height: 40px;
+}
+
+.illustration {
+  position: absolute;
+  padding-left: 6cm;
+  padding-top: 5cm;
+}
```

Comment:
- [ ] Kindly make sure to remove this code as the concerned image will be used as **backrounound image** in the home section as required in the project. I pointed out this in the previous comments.